fastlane/README.md

```
@@ -0,0 +1,50 @@
+fastlane documentation
+----
+
+# Installation
```

Markdownlint (0.37.0)

Expected: setext; Actual: atx
Heading style

(MD003, heading-style)